docs/developer/release-cycle.md

```diff
@@ -18,6 +18,8 @@ A release cycle can be split up into the following phases:
         * Providers can start adopting the new release based on the beta releases
     * All changes that impact providers' adoption of the new release should be announced in the provider updates section
       of the office hours meeting notes and approved in the PR or issue by both approvers and key affected providers.
+        * Non-breaking dependency bumps (usually minor and patch versions) which don't require provider changes 
```

Comment:
I suggest some more limits and chancing "usually" to ~"only". For capi direct dependencies using semantic versioning we can only bump minor and patch version without approval. For indirect dependencies there are no limitations.

IMO if a dependency bumps a major version then that indicates there are breaking api changes. So even if it doesn't break for capi it might break for a provider. So I think we should be explicit about it during beta.